jvm/ml4ir-inference/src/test/resources/classification/feature_config_with_same_name.yaml

```diff
@@ -0,0 +1,155 @@
+query_key:
+  name: query_key
+  node_name: query_key
+  trainable: false
+  dtype: string
+  log_at_inference: true
+  feature_layer_info:
+    type: string
+    shape: null
+  serving_info:
+    name: queryId
+    required: false
+  default_value: 0
+  tfrecord_type: context
+label:
+  name: entity_id
+  node_name: entity_id
+  trainable: false
+  dtype: string
+  shape:
+    - 1
+    - null
+  log_at_inference: true
+  preprocessing_info:
+    - fn: one_hot_vectorize_label
+  feature_layer_info:
+    type: numeric
+    fn: categorical_indicator_with_vocabulary_file
+    args:
+      vocabulary_file: ml4ir/applications/classification/tests/data/configs/vocabulary/entity_id.csv
+  serving_info:
+    name: entity_id
+    required: false
+  default_value: ""
```

Comment:
minor: default_value should be under serving_info.

Reply:
addressed in my update to this branch, @Ullimague !